src/main/java/hudson/plugins/plot/CSVSeries.java

```diff
@@ -38,11 +38,11 @@
  * @author Allen Reese
  */
 public class CSVSeries extends Series {
-    private static final Logger LOGGER = Logger.getLogger(CSVSeries.class.getName());
+    private static final transient Logger LOGGER = Logger.getLogger(CSVSeries.class.getName());
     // Debugging hack, so I don't have to change FINE/INFO...
-    private static final Level DEFAULT_LOG_LEVEL = Level.FINEST;
-    private static final Pattern PAT_SEMICOLON_ENCLOSURE = Pattern.compile("\"(.*?)\"");
-    private static final Pattern PAT_COMMA = Pattern.compile(",");
+    private static final transient Level DEFAULT_LOG_LEVEL = Level.FINEST;
```

Comment:
@MarkEWaite Isn't the `transient` redundant for `static` fields? They should not tamper with serialization?

Reply:
> @MarkEWaite Isn't the `transient` redundant for `static` fields? They should not tamper with serialization?

I've had bad experiences with the git plugin because I changed things related to serialization.  Keeping this code unmodified is more about me not wanting to take risks on a plugin that I do not actively use.